saleor/order/utils.py

```diff
@@ -78,7 +78,10 @@ def automatically_fulfill_digital_lines(order: Order):
             fulfillment=fulfillment, order_line=line, quantity=quantity
         )
         fulfill_order_line(order_line=line, quantity=quantity)
-    emails.send_fulfillment_confirmation.delay(order.pk, fulfillment.pk)
+    emails.send_fulfillment_confirmation_to_customer(
+        order, fulfillment, user=order.user
+    )
+    update_order_status(order)
```

Comment:
Do we have a test that checks whether an order has the right status after this operation?